<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface list members</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="interfaceField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Interface</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QListWidget" name="disableField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
